client/src/main/com/sinch/sdk/domains/voice/api/v1/adapters/WebHooksService.java

```diff
@@ -0,0 +1,68 @@
+package com.sinch.sdk.domains.voice.api.v1.adapters;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.sinch.sdk.core.exceptions.ApiMappingException;
+import com.sinch.sdk.core.http.AuthManager;
+import com.sinch.sdk.core.utils.MapUtils;
+import com.sinch.sdk.core.utils.databind.Mapper;
+import com.sinch.sdk.domains.voice.models.v1.svaml.SvamlControl;
+import com.sinch.sdk.domains.voice.models.v1.webhooks.VoiceWebhookEvent;
+import com.sinch.sdk.domains.voice.models.v1.webhooks.internal.WebhooksEventInternalImpl;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class WebHooksService implements com.sinch.sdk.domains.voice.api.v1.WebHooksService {
+  private static final Logger LOGGER = Logger.getLogger(WebHooksService.class.getName());
+
+  private final Map<String, AuthManager> authManagers;
+
+  public WebHooksService(Map<String, AuthManager> authManagers) {
+    this.authManagers = authManagers;
+  }
+
+  public boolean validateAuthenticationHeader(
+      String method, String path, Map<String, String> headers, String jsonPayload) {
+
+    // convert header keys to use case-insensitive map keys
+    Map<String, String> caseInsensitiveHeaders = MapUtils.getCaseInsensitiveMap(headers);
+
+    String authorizationHeader = caseInsensitiveHeaders.get("Authorization");
+
+    // no authorization required
+    if (null == authorizationHeader) {
+      return true;
```

Comment:
Shouldn't it be `false` if no authorization header is present?

Reply:
Indeed !
Fixed
Side note: not addressed with this PR, but a ticket is existing in queue to extend unit tests related to Voice/Verification about authentication (DEVEXP-404)